index.html

```diff
@@ -89,7 +96,8 @@ <h2 class="tonic1">Multi-Post Stories</h2>
         </ul>
         <button class="button" type="button" id="message">See project</button>
```

Comment:
- [ ] Kindly fix the button styling: Active, Hover, and Pressed 
<img width="1440" alt="Screenshot 2023-02-25 at 05 12 04" src="https://user-images.githubusercontent.com/81444338/221358769-8df6a18b-339b-4262-8234-1d4b5f0db053.png">

Reply:
Done.